src/Altinn.Platform/Altinn.Platform.Storage/Storage/Startup.cs

```diff
@@ -32,15 +35,34 @@ public Startup(IConfiguration configuration)
         public IConfiguration Configuration { get; }
 
         /// <summary>
-        /// configure database setttings for the service
+        /// configure database settings for the service
         /// </summary>
         /// <param name="services">the service configuration</param>        
         public void ConfigureServices(IServiceCollection services)
         {            
-            services.AddControllers().AddNewtonsoftJson();
+            services.AddControllers(config =>
+            {
+                AuthorizationPolicy policy = new AuthorizationPolicyBuilder()
+                    .AddAuthenticationSchemes(JwtCookieDefaults.AuthenticationScheme)
+                    .RequireAuthenticatedUser()
+                    .Build();
+                config.Filters.Add(new AuthorizeFilter(policy));
+            }).AddNewtonsoftJson();
             services.Configure<AzureCosmosSettings>(Configuration.GetSection("AzureCosmosSettings"));
             services.Configure<AzureStorageConfiguration>(Configuration.GetSection("AzureStorageConfiguration"));
-            services.Configure<GeneralSettings>(Configuration.GetSection("BridgeSettings"));
+            services.Configure<GeneralSettings>(Configuration.GetSection("GeneralSettings"));
+
+            GeneralSettings generalSettings = Configuration.GetSection("GeneralSettings").Get<GeneralSettings>();
+
+            services.AddAuthentication(JwtCookieDefaults.AuthenticationScheme)
+                .AddJwtCookie(JwtCookieDefaults.AuthenticationScheme, options =>
+                {
+                    options.ExpireTimeSpan = new TimeSpan(0, 30, 0);
+                    options.Cookie.Name = "AltinnStudioRuntime";
```

Comment:
could we get this from settings as well? I think that's what done for the app as well

Reply:
Done